build.gradle

```diff
@@ -116,7 +116,7 @@ allprojects {
     maven { url "https://hyperledger.jfrog.io/artifactory/besu-maven" }
     maven { url "https://hyperledger-org.bintray.com/besu-repo/" }
     maven { url "https://artifacts.consensys.net/public/maven/maven/" }
```

Comment:
Do we need this maven repo for anything?

Reply:
signers libraries would come from `https://artifacts.consensys.net/public/maven/maven/`

Reply:
👍